catalog/config-schema.json

```diff
@@ -64,7 +64,7 @@
     },
     "ssoProviders": {
       "type": "string",
-      "pattern": "^(google)|(\\$\\{\\w+\\})$",
+      "pattern": "^(google)?|(\\$\\{\\w+\\})$",
```

Comment:
Actually, the parens are wrong (both in the old and the new version): it's gonna be "^..." or "...$". Needs to be:
`^((google)?|\\$\\{\\w+\\})$`
or just
`^(|google|\\$\\{\\w+\\})$`